src/main/kotlin/com/cjbooms/fabrikt/configurations/PackagesConfig.kt

```diff
@@ -0,0 +1,23 @@
+package com.cjbooms.fabrikt.configurations
+
+import com.cjbooms.fabrikt.model.Destinations.clientPackage
+import com.cjbooms.fabrikt.model.Destinations.modelsPackage
+
+data class PackagesConfig(val packages: Packages) {
```

Comment:
Looks like this could be simplified to:
```
   class Packages(val base: String) {
        val models: String  = modelsPackage(base)
        val client: String  = clientPackage(base)
    }
```